core/types/gen_header_json.go

```diff
@@ -117,6 +119,7 @@ func (h *Header) UnmarshalJSON(input []byte) error {
 		StateLimit                      *hexutil.Uint64         `json:"stateLimit"                  gencodec:"required"`
 		StateUsed                       *hexutil.Uint64         `json:"stateUsed"                   gencodec:"required"`
 		Extra       					hexutil.Bytes  		 	`json:"extraData"          			gencodec:"required"`
+		SecondaryCoinbase               *common.Address         `json:"secondaryCoinbase"           gencodec:"required"`
```

Comment:
Can you do common.MixedCaseAddress here? If not I can do it

Reply:
done

Reply:
changed it